go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -173,6 +174,78 @@ func setupPIDFile(pidfile string) error {
 	return nil
 }
 
+// ovnkubeRunMode object stores the run mode of the ovnkube
+type ovnkubeRunMode struct {
+	networkControllerManager bool // network controller manager (--init-network-controller-manager) is enabled
+	clusterManager           bool // cluster manager (--init-cluster-manager) is enabled
+	node                     bool // node (--init-node) is enabled
+	cleanupNode              bool // cleanup (--cleanup-node) is enabled
+}
+
+// determineOvnkubeRunMode determines the run modes of ovnkube
+// based on the init flags set.  It is possible to run ovnkube in
+// multiple modes.  Allowed multiple modes are:
+//   - network controller manager + cluster manager
+//   - network controller manager + node
```

Comment:
I think the code allows it.   In the present upstream master code, the code allows it to run ovnkube in both "--init-master" and "--init-node".

The same is done here.  Except that both cluster manager mode and node mode is not supported since cluster manager doesn't require any DB connection was "node" mode does.

I think technically it is possible.  But I'd say its a reasonable restriction to begin with. Let me know if you think otherwise.

Reply:
i think renaming this to master + node makes more sense?
master here is init-master which is network controller manager + cluster-manager + node ?

Reply:
Ack.  Actually I missed documenting - master (network controller manager + cluster manager) + node 

I'll add it.